### PR TITLE
fs-1625-adding-error-handling-for-csrf-expiry

### PR DESCRIPTION
**Pull request by gio-karunakaran**

### CODE REVIEW

File: app/default/routes.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
 from flask import request
 from flask import url_for
 from flask_wtf import FlaskForm
-from fsd_utils.authentication.decorators import login_required
+from fsd_utils.authentication.decorators import login_required, login_requested
```
Comment (harryyo) [inline app/default/routes.py:20]: Has black run on this and allowed comma separated imports?

Comment (harryyo) [reply app/default/routes.py:20]: 😮

**Review by harryyo — APPROVED** [review]
Great - only my shock horror that black has allowed comma separated imports! (I have nothing against them...!)

File: app/default/routes.py (hunk 1 above)
Comment (gio-karunakaran) [reply app/default/routes.py:20]: it did! I'm surprised too! let me separate them out for consistency's sake at least 👍🏾

**Pull request merged**